src/test/java/com/google/sps/RecommendedCareerPathTest.java

```diff
@@ -0,0 +1,55 @@
+package com.google.sps;
+
+import com.google.sps.data.ProcessCareerQuizResults;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import com.google.sps.data.CareerQuestionChoice;
+import com.google.sps.data.CareerQuestionAndChoices;
+import java.util.List;
+import java.util.Arrays;
+
+
+/** Tests the CareerQuiz servlet and its interactions with CareerQuestionDatabase*/
+@RunWith(JUnit4.class)
+public final class RecommendedCareerPathTest {
+
+  private static final String CAREER_1 = "Web Developer";
+  private static final String CAREER_2 = "Program Manager";
+  private static final String CAREER_3 = "Software Engineer";
+  private static final String CHOICE_1 = "choice1";
+  private static final String CHOICE_2 = "choice2";
+  private static final String CHOICE_3 = "choice3";
+  private static final String CHOICE_4 = "choice4";
+  private static final String CHOICE_5 = "choice5";
+  
+  @Test
+  public void outputsMostFrequentCareerPath_2choices() {
+    List<CareerQuestionChoice> userChoices = Arrays.asList(
+      new CareerQuestionChoice(CHOICE_1, CAREER_1),
+      new CareerQuestionChoice(CHOICE_2, CAREER_2),
+      new CareerQuestionChoice(CHOICE_3, CAREER_2),
+      new CareerQuestionChoice(CHOICE_4, CAREER_3)
+    );
+    String recommendedCareerPath = ProcessCareerQuizResults.getRecommendedCareerPath(userChoices);
+    Assert.assertEquals(recommendedCareerPath, CAREER_2);
```

Comment:
So I would structure tests to be more like:
```
  @Test
  public void outputsMostFrequentCareerPath_2choices() {
    List<CareerQuestionChoice> userChoices = Arrays.asList(
      new CareerQuestionChoice(CHOICE_1, CAREER_1),
      new CareerQuestionChoice(CHOICE_2, CAREER_2),
      new CareerQuestionChoice(CHOICE_3, CAREER_2),
      new CareerQuestionChoice(CHOICE_4, CAREER_3)
    );

    String expected = CAREER_2;

    String result = ProcessCareerQuizResults.getRecommendedCareerPath(userChoices);

    Assert.assertEquals(result, expected);
  }
```

Reply:
You can leave it as is if you feel more comfortable.

Reply:
I like the clarity of this structure, changed.